.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches:
+     - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Run gradle build
+        run: ./gradlew build
+
+      - name: Run Delta Coverage check
+        if: ${{ github.event_name == 'pull_request' && ( success() || failure() ) }}
+        continue-on-error: ${{ contains(github.event.pull_request.labels.*.name, 'suppress-delta-coverage') }}
+        run: ./gradlew deltaCoverage -PdiffBase="refs/remotes/origin/$GITHUB_BASE_REF"
```

Comment:
NIT

U can consult with @ajax-malov-n how to improve reporting of your CI

It's optional

Reply:
done